hooks/post_build

```diff
@@ -3,17 +3,18 @@
 if [ -n "${BUILD_DOCS}" ]; then
 	set -e
 	DOCS_IMAGE=${DOCS_IMAGE:-${IMAGE_NAME}-docs}
+	# shellcheck disable=SC2046
 	docker run \
 		--entrypoint '' \
 		--privileged \
 		-e DOCKER_GITCOMMIT=$(git rev-parse --short HEAD) \
-		-v $(pwd)/docs/yaml/docs:/docs \
+		-v "$(pwd)"/docs/yaml/docs:/docs \
```

Comment:
better to have `-v "$(pwd)/docs/yaml/docs:/docs"`

`$(pwd)` may contain "space(s)", e.g. `-v /path/space space/docs/yaml/docs:/docs`